pylti/common.py

```diff
@@ -218,7 +217,7 @@ def post_message(consumers, lti_key, url, body):
     """
     content_type = 'application/xml'
     method = 'POST'
-    (response, content) = _post_patched_request(
+    (_, content) = _post_patched_request(
```

Comment:
I recommend `__` instead of `_`. The later is usually used for `gettext`.

http://docs.python-guide.org/en/latest/writing/style/#create-an-ignored-variable